src/PlaywrightSharp.Abstractions/IFrame.cs

```diff
@@ -1,94 +1,104 @@
 ﻿using System.Text.Json;
-using System.Threading.Tasks;
```

Comment:
What happened here? I pushed an .editorconfig Is your IDE using that?

Reply:
hmm, not sure what happened here

Reply:
If we fix that this would be ready to go 🚀